src/frontends/tensorflow_common/include/openvino/frontend/tensorflow/decoder.hpp

```diff
@@ -5,13 +5,13 @@
 #pragma once
 
 #include "openvino/core/any.hpp"
-#include "openvino/frontend/tensorflow/visibility.hpp"
+#include "openvino/frontend/exception.hpp"
 
 namespace ov {
 namespace frontend {
 namespace tensorflow {
 
-class TENSORFLOW_API DecoderBase {
+class DecoderBase {
```

Comment:
Not sure that this is the right change for OVTF? How will they inherit their implementation of `Decoder` from the base `Decoder` class.

Reply:
@jane-intel we need to have export macro here, because these headers will be a part of TF public API and we need to export these headers from the frontend.

Reply:
If we export headers here, they will be exported in both TF and TF Lite FEs => multiple symbols

Reply:
If we don't do that, will it be accessible for OVTF?

Reply:
As I understood we will include this headers to both targets. Looks like TF and TF lite are two different frontends and we shouldn't have problems with multiple symbols